### DISCUSSION TITLE
【2023.12.17】1966720 Lethal Company 致命公司

### DISCUSSION THREAD

**Discussion by HUMHIHello** [pjy612/SteamManifestCache#71] [Share]
appid 1966720
keys
1966721:b79233b678dae6d3f928ff9c6c7674d559427776a1337dee238773a4fec60949
[【2023.12.17】1966720 Lethal Company 致命公司.zip](https://github.com/pjy612/SteamManifestCache/files/13694747/2023.12.17.1966720.Lethal.Company.zip)

参与内测https://t.me/U_U2033
或
hihello_0@163.com

(没办法 有一个很老的游戏就咱这个项目有 已经连续分享骚扰了三次了  希望管理员不要骂我qwq)

**Comment by pjy612** [comment]
抱歉 记错了 你似乎一直没留 内测码
但是 你那个报错 要修。。。 我可能要 改不少东西。。。

**Reply by HUMHIHello** [reply]
啊 我已经内测了吗
我以为之前分享的不合规呢
没事
不着急 我估计是个例没必要因为我那么麻烦

**Reply by HUMHIHello** [reply]
能有你们无偿付出已经很好了
再挑三拣四就不知好歹了

**Reply by pjy612** [reply]
哦 不对，记错了。。。
你是 因为一开始就用不了。然后 就一直没留code吧。。。

**Reply by HUMHIHello** [reply]
是的 我只留了邮箱和telegram
